src/NUnitFramework/testdata/TestFixtureSourceData.cs

```diff
@@ -216,6 +216,32 @@ private static IEnumerable MyData()
             new object[] { 12, 2, 6 } };
     }
 
+    [TestFixture]
+    [TestFixtureSource(nameof(MyData))]
+    public class TestFixtureSourceMayUseParamsArguments
+    {
+        public TestFixtureSourceMayUseParamsArguments(params int[] parameters)
+        {
+            Parameters = parameters;
+        }
+
+        public int[] Parameters { get; }
+
+        [Test]
+        public void Test()
+        {
+            Assert.That(Parameters, Is.Not.Null);
+            for (int i = 0; i < Parameters.Length; i++)
+                Assert.That(Parameters[i], Is.EqualTo(i + 1));
+        }
+        public static IEnumerable MyData()
```

Comment:
Please make the `MyData` method private in order not to raise `NUnit1028`: Only test methods should be public.